src/components/Wizard/SpotlightWizard/SpotlightScreensInput/LogoAndTitleScreen.tsx

```diff
@@ -21,6 +22,11 @@ function LogoAndTitleScreen({ controlPrefix, register }: ScreenComponentProps) {
         controlPrefix={`${controlPrefix}.logo`}
         register={register}
       />
+      <SpotlightDismissButtonInput
+        controlPrefix={`${controlPrefix}.dismissButton`}
+        register={register}
+        required
```

Comment:
Your component doesn't accept `required` prop and it doesn't need it. There shouldn't be a case where the dismiss button is required.